ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -234,6 +234,52 @@
    */
   CompletableFuture<ServerInfo> serverInfo();
 
+  /**
+   * Creates a connector.
+   *
+   * <p>If a non-200 response is received from the server, the {@code CompletableFuture} will be
+   * failed.
+   *
+   * @param connectorName name of the connector
+   * @param isSource true if the connector is a source connector, false if it is a sink connector
+   * @param properties connector properties
+   * @return result of connector creation
+   */
+  CompletableFuture<CreateConnectorResult> createConnector(
```

Comment:
@colinhicks do you want to sanity-check these new Java client interfaces for managing connectors, from the user/product perspective?